README.md

```diff
@@ -1196,13 +1156,12 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
 
 * Avoid needless metaprogramming.
 
-* Never start a method with `get_`.
+* Avoid starting method names with `get_`.
 
-* Never use a `!` at the end of a method name if you have no equivalent method
-  without the bang. Methods are expected to change internal object state; you
-  don't need a bang for that. Bangs are to mark a more dangerous version of a
-  method, e.g. `save` returns a `bool` in ActiveRecord, whereas `save!` will
-  throw an exception on failure.
+* Avoid ending method names with `!` when there is no equivalent method without
+  the bang. Bangs are to mark a more dangerous version of a method, e.g. `save`
+  returns a `bool` in ActiveRecord, whereas `save!` will throw an exception on
+  failure.
```

Comment:
Might be worth mentioning the mutation use case as well (e.g. `map` vs `map!`).

Also "`bool`" should probably just be "boolean"

Reply:
Replaced `bool` with "boolean". I'm not mentioning the mutation use case because that's the one we are encouraging people to avoid.